### ISSUE TITLE
solution notebook for `3_keras_sequential_api_vertex.ipynb` has really bad performance when run as is

### ISSUE THREAD

**Issue by sanjanalreddy** [GoogleCloudPlatform/asl-ml-immersion#345]
The solution notebook provides a solution where the loss curves don't converge for training and validation and the loss is pretty high even after 20 epochs. See loss curves below for reference:

<img width="742" alt="Screenshot 2023-05-17 at 1 57 20 PM" src="https://github.com/GoogleCloudPlatform/asl-ml-immersion/assets/10604151/2603e0d7-bcb4-4495-8227-13c9b50c7e20">

Suggestions to fix this:
- add learning rate to the optimizer
- add steps_per_epoch to the model.fit method

**Comment by takumiohym** [comment]
For this notebook, I think it is natural that the model doesn't converge since it just gets latitude/longitude without any feature engineering.
Did you get better result with larger learning rate?

For other notebooks (`4_keras_functional_api.ipynb`, `building_production_ml_systems/.../1_training_at_scale_vertex.ipynb`, `building_production_ml_systems/.../2_hyperparameter_tuning_vertex.ipynb`), I usually specify larger `NUM_TRAIN_EXAMPLES` to converge.

**Comment by sanjanalreddy** [comment]
Yes, I got better performance using learning rate and also re-introducing steps_per_epoch. I will add a PR to solve this. I agree that we don't get the best performance but we can do better than what we have in the solutions notebook.
